extensions/sql-database-projects/src/dialogs/createProjectFromDatabaseDialog.ts

```diff
@@ -140,35 +146,35 @@ export class CreateProjectFromDatabaseDialog {
 				.withItems([this.sdkStyleCheckbox, sdkLearnMore], { CSSStyles: { flex: '0 0 auto', 'margin-right': '10px' } })
 				.component();
 
+			const settingsFormSection = view.modelBuilder.groupContainer().withLayout({
+				header: constants.createProjectSettings,
+				collapsible: false,
+				collapsed: false
+			}).withItems([folderStructureRow, this.includePermissionsCheckbox, sdkFormComponentGroup]).component();
+
 			this.formBuilder = <azdataType.FormBuilder>view.modelBuilder.formContainer()
 				.withFormItems([
 					{
-						title: constants.sourceDatabase,
+						title: '',
```

Comment:
This originally used a `formContainer` to have the titles of the section, but now the header of the `groupContainer` is used for the titles. It would be ideal to switch to using something else like `divContainer` instead since this `title` isn't needed anymore, but when I tried to switch it didn't look nice, and at this point it isn't worth taking the time to switch and make it look pretty when it won't make any functional difference.